looprpc/client.proto

```diff
@@ -740,6 +745,105 @@ message LsatToken {
     string storage_name = 8;
 }
 
+message LoopOutStats {
+    /*
+    Number of currently pending loop outs.
+     */
+    uint64 pending_count = 1;
+
+    /*
+    Number of succeeded loop outs.
+     */
+    uint64 success_count = 2;
+
+    /*
+    Number failed loop outs.
+     */
+    uint64 fail_count = 3;
+
+    /*
+    The sum of all pending loop out amounts.
+     */
+    int64 sum_pending_amt = 4;
+
+    /*
+    The sum of all succeeded loop out amounts.
+     */
+    int64 sum_succeeded_amt = 5;
+}
+
+message LoopInStats {
```

Comment:
Since `LoopOutStats` & `LoopInStats` share the same fields, perhaps create a generic `LoopStats` field and then have `GetInfoResponse` contain the following:
```proto
LoopStats loop_out_stats = 7;

LoopStats loop_in_stats = 8;
```

Reply:
I like it. I have seen this pattern in the code but didn't connect the dots. I will add your suggestion.